TheHiveV5/action_add_comment.json

```diff
@@ -0,0 +1,61 @@
+{
+  "uuid": "f936765f-4b37-46c8-9801-701fc11385ea",
+  "name": "Add a comment in The Hive v5",
+  "description": "Action to add a comment in The Hive v5",
+  "docker_parameters": "thehive_add_comment",
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "alert_id": {
+        "description": "The id of the alert you want to add a comment to",
+        "default": "~4231176",
```

Comment:
It's a default value or an example? If it's an example, can you remove this field?

```suggestion
```